Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.yml

```diff
@@ -0,0 +1,2272 @@
+        \ only."
+      isArray: false
+      name: block_period
+      required: false
+    - auto: PREDEFINED
+      description: "Select which severity level the FortiWeb appliance will use\
+        \ when a blacklisted IP address attempts to connect to your web servers:LowMediumHighSupports\
+        \ API version 2 only."
+      isArray: false
+      name: severity
+      predefined:
+      - Low
+      - Medium
+      - High
+      - Info
+      required: false
+    - auto: PREDEFINED
+      description: "enable Ignore X-Forwarded-For so that the IP addresses will\
+        \ be scanned at the TCP layer instead of HTTP layer.Supports API version 2\
+        \ only."
+      isArray: false
+      name: ignore_x_forwarded_for
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: "Select which trigger, if any, that the FortiWeb appliance will\
+        \ use when it logs and/or sends an alert email about a blacklisted IP address\u2019\
+        s attempt to connect to your web serversSupports API version 2 only."
+      name: trigger_policy
+      required: false
+    deprecated: false
+    description: Update IP List.
+    execution: false
+    name: fortiwebvm-ip-list-group-update
+    outputs:
+    - contextPath: FortiwebVM.IpListGroup.id
+      description: IP List Name.
+      type: String
+  - arguments:
+    - description: IP list group name.
+      isArray: false
+      name: name
+      required: true
+    deprecated: false
+    description: Supports API versions 1 & 2.
+    execution: false
+    name: fortiwebvm-ip-list-group-delete
+  - arguments:
+    - description: IP List Name.
+      name: name
+    - description: The page number of the results to retrieve.
+      isArray: false
+      name: page
+      required: false
+    - description: The maximum number of records to retrieve per page.
+      isArray: false
+      name: page_size
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of records to retrieve.
+      isArray: false
+      name: limit
+      required: false
+    deprecated: false
+    description: Supports API versions 1 & 2.
+    execution: false
+    name: fortiwebvm-ip-list-group-list
+    outputs:
+    - contextPath: FortiwebVM.IpListGroup._id
+      description: IP list id.Supports API version 1 only.
+      type: String
+    - contextPath: FortiwebVM.IpListGroup.id
+      description: IP list index.
+      type: Number
+    - contextPath: FortiwebVM.IpListGroup.name
+      description: IP list name.
+      type: String
+    - contextPath: FortiwebVM.IpListGroup.ip_list_count
+      description: IP list members number.
+      type: Number
+    - contextPath: FortiwebVM.IpListGroup.results.can_delete
+      description: Can delete flag.Supports API version 1 only.
+      type: Boolean
+    - contextPath: FortiwebVM.IpListGroup.results.can_view
+      description: Supports API version 2 only.
```

Comment:
Please make sure all items have a description that shows what they are, e.g. `Whether <...> can view <...> .This value is only avaialble when using API v2` (here, and in all other outputs)